tests/kafkatest/services/kafka/kafka.py

```diff
@@ -602,12 +602,6 @@ def start_minikdc_if_necessary(self, add_principals=""):
                         nodes_for_kdc += other_service.nodes
                     self.minikdc = MiniKdc(self.context, nodes_for_kdc, extra_principals = add_principals)
                     self.minikdc.start()
-        else:
```

Comment:
Should we add a comment to clarify why it's important not to unassign?

Reply:
Will do,thanks!